int8   a_button
int8   b_button
int8   x_button
int8   y_button

int8   dpad_left
int8   dpad_right
int8   dpad_up
int8   dpad_down

int8   back_button
int8   center_button
int8   start_button

int8   left_joy_click
int16  left_joy_x
int16  left_joy_y

int8   right_joy_click
int16  right_joy_x
int16  right_joy_y

uint8   left_bump
uint8   right_bump
uint8   left_trig
uint8   right_trig
